packages/dropdowns.next/demo/menu.stories.mdx

```diff
@@ -0,0 +1,19 @@
+import { Meta, ArgsTable, Canvas, Story } from '@storybook/addon-docs';
+import { Menu, Item, ItemGroup } from '@zendeskgarden/react-dropdowns.next';
+import { MenuStory } from './stories/MenuStory';
+
+<Meta
+  title="Packages/Dropdowns.Next/Menu"
+  component={Menu}
+  subcomponents={{ Item, 'Item.Meta': Item.Meta, ItemGroup }}
+/>
+
+# API
+
+<ArgsTable />
+
+# Demo
+
+<Canvas>
+  <Story name="Menu">{args => <MenuStory {...args} />}</Story>
```

Comment:
Prior to merge, I plan to rename this file so the story doesn't leak into production.